### ISSUE TITLE
Recommend adding info on how to run the code to your README.md

### ISSUE THREAD

**Issue by dwjohnston** [emmalearnscode/guide-to-HF#36]
Instructions for 'how to get this to run after I check it out' should be included in the main README. It's the first place people typically look.

**Comment by Liatmoss** [comment]
I'd be happy to add more information to the readme. Could this be assigned to me please?

**Comment by emmalearnscode** [comment]
I'm not entirely sure what needs to be added @Liatmoss . @dwjohnston Could you elaborate so I know what should be included?

**Comment by dwjohnston** [comment]
(ugh, hang on I have the switch computer. This comment is still in progress). 

, so to be honest I think I created this issue before I realised that this is  an HTML only project (ie. no npm commands) , and that the intention is mostly for people to create 'standalone' static pages. 

That said it sounds like that needs to be clarified. What I would suggest is there are three ways someone might 'run the code'. 

- Using the Live Server plugin for VSCode (That's what I do). 
- Use the [serve ]([url](https://www.npmjs.com/package/serve))package. 
- "manually navigate to your html file and open it in your browser".
